### PR TITLE
Add 選項 parameter

### PR DESCRIPTION
**Pull request by ayaka14732**
Add 選項 parameter to each schema so that qieyun-autoderiver can read the options.

### CODE REVIEW

**Review by untunt — APPROVED** [review]

File: unt_j.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
  * 之前的 unt 切韵朗读音推导方案已归档，请查看 https://github.com/nk2028/qieyun-examples/blob/main/unt.js
  */
 
-const is = (x) => 音韻地位.屬於(x);
+const is = (x) => 音韻地位.屬於(x.replace(/　/g,''));
```
Comment (graphemecluster) [inline unt_j.js:13]: Missing space after comma (nitpicking)

File: chiangxhua.js [hunk 2]
```diff
@@ -174,18 +174,18 @@ let 唇化 = 唇化規則();
...
 
-if (聲調用調類表示) {
+if (選項.聲調用調類表示) {
   音位 = 聲母 + 介音 + 唇化 + 韻核 + 韻尾 + 聲調;
 } else {
   音位 = 聲母 + 介音 + 唇化 + 韻核 + 聲調 + 韻尾;
 }
 
-if (顯示音位 && !顯示音值) return 音位;
+if (選項.顯示 == "音位") return 音位;
```
Comment (graphemecluster) [inline chiangxhua.js:188]: I should use single quote

File: chiangxhua.js [hunk 3]
```diff
@@ -281,6 +281,6 @@ if (聲調用調類表示) {
 音值 = 音值.replace('ɥ̈ʉ', 'ʉ');
 音值 = 音值.replace('wu', 'u');
 
-if (!顯示音位 && 顯示音值) return 音值;
+if (選項.顯示 == "音值") return 音值;
```
Comment (graphemecluster) [inline chiangxhua.js:284]: Same here

File: chiangxhua.js (hunk 2 above)
Comment (graphemecluster) [reply chiangxhua.js:188]: And also strictly equal (`===`)

**Review by graphemecluster — APPROVED** [review]

**Review by syimyuzya — APPROVED** [review]

**Comment by graphemecluster** [comment]
build.py
https://github.com/nk2028/qieyun-examples/compare/%E9%81%B8%E9%A0%85...graphemecluster:%E9%81%B8%E9%A0%85

**Comment by graphemecluster** [comment]
`build.py` updated, please add the commit again.

**Comment by ayaka14732** [comment]
@graphemecluster Do you have the permission to commit to this branch directly?

**Comment by graphemecluster** [comment]
@ayaka14732 Of course I don’t, that’s why I ask you.
Anyway, I think I won’t modify the file again unless there’re any critical issues.

**Comment by ayaka14732** [comment]
係呢度出咗問題（multiline string 唔可以縮進）：

```javascript
  if (is(`幫組 一二三四等 \
  或 端組 一四等 \
  或 知組 二三等 \
  或 精組 一三四等 \
  或 莊組 二三等 \
  或 章組 三等 \
  或 見溪疑母 一二三四等 \
  或 羣母 二三等 \
  或 影曉母 一二三四等 \
  或 匣母 一二四等 \
  或 云以母 三等 \
  或 來母 一二三四等 \
  或 日母 三等`)) {
    隔音符號 = '';
  }
```

**Comment by graphemecluster** [comment]
我都知喺呢度出問題，但係就係諗唔到點解，原來係縮進咗會出問題。
（不過調返轉嚟講，將 `qieyun-js` 嗰邊嘅 `split(' ')` 改做 `split(/\s+/)` 都唔係唔得）

**Comment by graphemecluster** [comment]
噉前面嘅 `'  ' + ` 都可以 delete 埋

**Comment by ayaka14732** [comment]
主要係我唔想支援咁多空格方法，所以凈係支援一個&nbsp;` `

File: build.py [hunk 4]
```diff
@@ -1,7 +1,59 @@
...
+ * @param {string} 字頭 要查詢的字頭
+ * @param {Object=} 選項 選項（可選）
+ * @return {{音韻地位: Qieyun.音韻地位, 解釋: string, 擬音: string}[]} 音韻地位、解釋、音韻地位對應的擬音
+ *//**
+ * 查詢字頭的擬音。
+ * @param {string[]} schema 推導方案陣列
+ * @param {string} 字頭 要查詢的字頭
+ * @param {Object=} 選項 選項（可選）
+ * @return {{音韻地位: Qieyun.音韻地位, 解釋: string, 擬音: string[]}[]} 音韻地位、解釋、音韻地位對應的擬音陣列
+ */
```
[the remark this answers is not in the document]
Comment (graphemecluster) [reply build.py]: 我想寫嘅意思係 `schema` 傳入 Array `擬音` 就出 Array，傳入 string 就出 string

[the remark this answers is not in the document]
Comment (ayaka14732) [reply build.py]: 我明～但係 VSCode 識別唔到，所以我改咗

[the remark this answers is not in the document]
Comment (graphemecluster) [reply build.py]: 我知啦，正確嘅做法應該係要開個 `.d.ts` 嘅 TypeScript definition file 先可以多載

**Pull request merged**